.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "editor.detectIndentation": false,
+  "editor.tabSize": 2
```

Comment:
2?

Ew.

Reply:
Ha Ha. Whoops. Didn't mean to commit that.

Reply:
You and your 2 space tabs *shudders